source/help/registerhelp.md

```diff
@@ -1,48 +1,46 @@
 # Author Registration
 
 To begin registration, visit the [registration form](https://arxiv.org/user/register).
-First, we will ask for your e-mail and you will get to pick a user name
-and password. Your user name must be unique and cannot be changed.  
-Recommendations for arXiv password:
 
--  8 characters or more in length
+Your username must be unique and cannot be changed.  
 
-- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
+arXiv registration also asks for your name and your Organization. If you are unaffiliated you may enter "Indepdendent" as your organization.
+
+Select which archives you are most likely to submit to. This will filter the category options available if you submit an article in the future. 
```

Comment:
Suggestion: Remove "in the future". I think it filters options immediately after successful account creation.